app/components/FrameworkSelect.tsx

```diff
@@ -0,0 +1,118 @@
+import { Fragment } from 'react'
+import { Listbox, Transition } from '@headlessui/react'
+
+import { HiCheck, HiChevronDown } from 'react-icons/hi'
+
+import vueLogo from '../images/vue-logo.svg'
+import reactLogo from '../images/react-logo.svg'
+import solidLogo from '../images/solid-logo.svg'
+import svelteLogo from '../images/svelte-logo.svg'
+import angularLogo from '../images/angular-logo.svg'
+import { Form, useMatches } from '@remix-run/react'
+import { useNavigate } from '@remix-run/react'
+
+// TODO: we could potentially pass available frameworks trough config
+const frameworks = [
+  { label: 'React', value: 'react' },
+  { label: 'Solid', value: 'solid' },
+  { label: 'Vue', value: 'vue' },
+  { label: 'Svelte', value: 'svelte' },
+  { label: 'Angular', value: 'angular' },
```

Comment:
we won't really have angular in tanstack/query because it's in a community adapter:

https://tanstack.com/query/v4/docs/community/angular-query

is there a way to make this customizable ? Maybe we could bind it to the structure that is listed under `adapters` ?

<img width="228" alt="Screenshot 2022-12-08 at 07 49 53" src="https://user-images.githubusercontent.com/1021430/206378217-1a4ff37f-4544-418d-a09c-d3c0c40c82a8.png">

Reply:
Available options in the switcher are now inherited based on sections from the docs config